ipn/ipnlocal/local.go

```diff
@@ -1939,6 +1941,10 @@ func (b *LocalBackend) ResendHostinfoIfNeeded() {
 func (b *LocalBackend) WatchNotifications(ctx context.Context, mask ipn.NotifyWatchOpt, onWatchAdded func(), fn func(roNotify *ipn.Notify) (keepGoing bool)) {
 	ch := make(chan *ipn.Notify, 128)
 
+	randomBytes := make([]byte, 16)
+	rand.Read(randomBytes)
+	sessionID := hex.EncodeToString(randomBytes)
```

Comment:
sent https://github.com/tailscale/tailscale/pull/9237 which this could use

Reply:
Approved, will use that if it goes in first. Otherwise, I'll send a follow up. Thanks!